package.yaml

```diff
@@ -1,5 +1,5 @@
 name:                concordium-client
-version:             7.0.1
+version:             7.0.2
```

Comment:
Will this be reverted to 7.0.1?

Reply:
This should definitely stay. The artifacts has already been released, and we do not want to overwrite them